cumulus/parachains/integration-tests/emulated/common/src/lib.rs

```diff
@@ -49,6 +51,25 @@ pub const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
 
 type AccountPublic = <MultiSignature as Verify>::Signer;
 
+// This asset is added to AH as Asset and reserved transfer between Parachain and AH
+pub const RESERVABLE_ASSET_ID: u32 = 1;
+// This asset is added to AH as ForeignAsset and teleported between Penpal and AH
+pub const TELEPORTABLE_ASSET_ID: u32 = 2;
+
+pub const PENPAL_ID: u32 = 2000;
+pub const ASSETS_PALLET_ID: u8 = 50;
+
+parameter_types! {
+	pub PenpalTeleportableAssetLocation: xcm::v3::Location
+		= xcm::v3::Location::new(1, [
+				xcm::v3::Junction::Parachain(PENPAL_ID),
+				xcm::v3::Junction::PalletInstance(ASSETS_PALLET_ID),
+				xcm::v3::Junction::GeneralIndex(TELEPORTABLE_ASSET_ID.into()),
+			]
+		);
+	pub PenpalSiblingSovereigAccount: AccountId = Sibling::from(PENPAL_ID).into_account_truncating();
+}
+
```

Comment:
```suggestion
```

I think this stuff is penpal-specific, so I would move it to the chain definition for `emualted/chains/parachains/testing/penpal` and where needed import `penpal-emulated-chain` dependency

Reply:
The reason I added in `common` is because I do not want emulated chain's declarations to depend on other chain. Emulated Chain's declarations should be independent and not pull any other runtime.

I want other teams to be able to import for example `asset-hub-rococo-emulated-chain` without importing unrelated `penpal-runtime`.